interfaces/mount/backend.go

```diff
@@ -58,7 +58,7 @@ func (b *Backend) Setup(snapInfo *snap.Info, confinement interfaces.ConfinementO
 	}
 	content := deriveContent(spec.(*Specification), snapInfo)
 	// synchronize the content with the filesystem
-	glob := fmt.Sprintf("%s.fstab", interfaces.SecurityTagGlob(snapName))
+	glob := fmt.Sprintf("snap.%s.*fstab", snapName)
```

Comment:
Aha, thanks. One more (silly) question - do we have a tesw for that? To prevent that when some next guy comes along and "fixes" it something breaks?